integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/client/ElasticsearchClientFactoryImplIT.java

```diff
@@ -1008,6 +1008,19 @@ public void clientInstance() throws IOException {
 		}
 	}
 
+	@Test
+	public void maxKeepAliveNegativeValue() {
```

Comment:
To test "positive" cases, you may be able to leverage `ElasticsearchHttpClientConfigurer` to insert some sort of spy on connection opening/closing. That should allow you to test that connections are kept alive as desired.

Reply:
I haven't found any method other than "proxying" the connection pool manager to check which connections are released.

Reply:
Works for me, thanks